test/integration/dialects/postgres/query-interface.test.js

```diff
@@ -173,6 +173,34 @@ if (dialect.match(/^postgres/)) {
             expect(res[0].my_func).to.be.eql('second');
           });
       });
+
+      it('produces an error when options.variables is missing expected parameters', function() {
+        const body = 'return 1;';
+        return Promise.all([
```

Comment:
I don't think `Promise.all` makes sense here, just remove it, calling the three `expect`s in sequence normally. The throws shall happen synchronously anyway...